Gaijinizer patch/Enemies.txt

```diff
@@ -1397,18 +1372,18 @@ function is activated.#MTLed
 だが、躯はそれを自身の髪の毛で固定し
 自身の意志で、腰振りを永遠に続けるのだ。
 > CONTEXT: Enemies/549/BookDescription/2
-Maximum speed for her 'waist swing feature'
-Originally, long-term use is a crisis of life
-The Hug feature is also released in a fixed amount of time,
-and is now ready to go.
-But Mukuro fixed it with his own hair
-With your will, keep the waist swing forever.#MTLed
+The maximum speed setting for waist swing. Since
+operating for a long time can be hazardous, a
+safety release timer was built into the doll so
+she would release the user after a set time. But
+Mukuro uses her hair to circumvent this, and
+keeps waist swing activated with sheer willpower.#RonIffyWording
 > END STRING
```

Comment:
Flow and rewording:
>_**Her Waist Swing at full power. Since performing
this for a long time can be hazardous, a safety
release timer was built into her body so she
would release the user after a set time. Mukuro
uses her hair to circumvent the timer, forcefully
keeping it activated for as long as she wants.**_